x-pack/plugins/enterprise_search/public/applications/app_search/cypress/support/commands.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login as baseLogin } from '../../../shared/cypress/commands';
+import { appSearchPath } from '../../../shared/cypress/routes';
+
+interface Login {
+  path?: string;
+  username?: string;
+  password?: string;
+}
+export const login = ({ path = '/', ...args }: Login = {}) => {
+  baseLogin({ ...args });
+  cy.visit(`${appSearchPath}${path}`);
```

Comment:
Quick explanation for the (new-ish) `login({ path })` param - actually hard-visiting URL links takes way more time for Kibana than it did in ent-search, so if possible we should try to only visit Kibana once and use SPA navigation from then on. So ideally this will help with that, e.g. `login({ path: '/engines/test/relevance_tuning' })`